__PROJECT NAME__/fastlane/Fastfile

```diff
@@ -0,0 +1,94 @@
+require("dotenv")
+Dotenv.load
+default_platform(:ios)
+
+platform :ios do
+  desc "Check, download and install certifications and provisioning profiles"
+  lane :download_certs_and_provisioning_profiles do
+    # Development Cert.
+    cert(
+      development: true,
+      username: "#{ENV['APPLE_ID']}",
+      output_path: "./code_signing/certificates/development",
+    )
+
+    # Distrubution Cert.
+    cert(
+      username: "#{ENV['APPLE_ID']}",
+      output_path: "./code_signing/certificates/distribution",
+    )
+
+    # Development Mobile Provisioning ( DEBUG )
+    sigh(
+      development: true,
+      username: "#{ENV['APPLE_ID']}",
+      app_identifier: "#{ENV['BUNDLE_ID_DEBUG']}",
+      output_path: "./code_signing/profiles/development",
+    )
+
+    # Ad-Hoc Mobile Provisioning ( STAGING )
+    sigh(
+      adhoc: true,
+      username: "#{ENV['APPLE_ID']}",
+      app_identifier: "#{ENV['BUNDLE_ID_STAGING']}",
+      output_path: "./code_signing/profiles/adhoc",
+    )
+
+    # AppStore Mobile Provisioning ( RELEASE )
+    sigh(
+      username: "#{ENV['APPLE_ID']}",
+      app_identifier: "#{ENV['BUNDLE_ID']}",
+      output_path: "./code_signing/profiles/appstore",
+    )
+  end
+
+  desc "setup_template"
+  lane :setup_template do
```

Comment:
how about changing its name to be more specific? perhaps something like `setup_app_on_developer_portal` and let's put in some comment on how it works